dart-impl/mpi/CMakeLists.txt

```diff
@@ -6,8 +6,6 @@ set(DASH_DART_IMPL_MPI_LIBRARY dart-mpi)
 
 set(DASH_DART_BASE_LIBRARY dart-base)
 
-include(${CMAKE_SOURCE_DIR}/CMakeExt/MPI.cmake)
-
```

Comment:
DART should work as standalone module and hence cmake exports it as its own package. So I guess this include is mandatory here. DASH should not directly require information on MPI, but the MPI implementation can be queried using the dash config interface. This is used in some unit tests which do not work with some MPI backends.